tau_bench/envs/airline/tasks_test.py

```diff
@@ -592,7 +592,7 @@
                 kwargs={
                     "reservation_id": "VA5SGQ",
                     "total_baggages": 1,
-                    "nonfree_baggages": 1,
+                    "nonfree_baggages": 0,
```

Comment:
The flight is economy, so the user should get one free bag. Requires https://github.com/sierra-research/tau-bench/pull/23.